packages/core/src/change_detection/scheduling/zoneless_scheduling.ts

```diff
@@ -66,7 +66,7 @@ export abstract class ChangeDetectionScheduler {
 /** Token used to indicate if zoneless was enabled via provideZonelessChangeDetection(). */
 export const ZONELESS_ENABLED = new InjectionToken<boolean>(
   typeof ngDevMode === 'undefined' || ngDevMode ? 'Zoneless enabled' : '',
-  {providedIn: 'root', factory: () => false},
+  {providedIn: 'root', factory: () => true},
```

Comment:
Should this refer to the `ZONELESS_BY_DEFAULT` const?

```suggestion
  {providedIn: 'root', factory: () => ZONELESS_BY_DEFAULT},
```